src/Data/Connection/UserConnectionResolver.php

```diff
@@ -148,10 +149,9 @@ public static function sanitize_input_fields( array $args, $source, array $all_a
 		 * @since 0.0.5
 		 * @return array
 		 */
-		$query_args = apply_filters( 'graphql_map_input_fields_to_wp_comment_query', $query_args, $args, $source, $all_args, $context, $info );
+		$query_args = apply_filters( 'graphql_map_input_fields_to_wp_comment_query', $query_args, $args, $this->source, $this->args, $this->context, $this->info );
```

Comment:
Any way we could centralize this in the abstract?

Reply:
ya, I think that's a good idea. 🤔

Reply:
Ok, so I've added a filter to the AbstractConnectionResolver for this. Just trying to think if we fully deprecate that filter, or include some deprecated filter functionality that calls that old filter from the new one. 

For example, I added `apply_filters( 'graphql_connection_query_args', $this->get_query_args(), $this )`

So we could have a `Deprecated.php` that does something like: 

```
add_filter( 'graphql_connection_query_args', function( $args, $connection ) {
    apply_filters( 'graphql_map_input_fields_to_wp_comment_query', $args, $connection->source, $connection->args, $connection->context, $connection->info );
}, 10, 2 );
```

Reply:
I think I'm in favor of removing and keeping clean, documenting the update path, then coming up with a better deprecation method after v1.0.0 when we're communicating that we're stable